lib/matplotlib/lines.py

```diff
@@ -436,11 +436,18 @@ def contains(self, mouseevent):
         :meth:`~matplotlib.lines.Line2D.set_pickradius` to view or
         modify it.
 
-        Returns *True* if any values are within the radius along with
-        ``{'ind': pointlist}``, where *pointlist* is the set of points
-        within the radius.
+        Parameters
+        ----------
+        mouseevent : `matplotlib.backend_bases.MouseEvent`.
 
-        TODO: sort returned indices by distance
+        Returns
+        -------
+        contains : bool
+            Whether any values are within the radius.
+        ``{'ind': pointlist}``
+            Where *pointlist* is the set of points within the radius.
+
+            TODO: sort returned indices by distance
```

Comment:
leave this out of the docstring? :)

Reply:
Well, it's been in there before. I didn't bother to decide if this should stay or be moved to a comment or be deleted completely.

Just checked: It's a comment by John D. Hunter from 2007. Anyone thinks this is still relevant?

Reply:
Let's just leave it in for now, didn't notice it was already there.